emission/exportdata/export_data.py

```diff
@@ -0,0 +1,50 @@
+# Standard imports 
+import logging
+
+# Our imports
+import emission.storage.pipeline_queries as epq
+import emission.storage.timeseries.abstract_timeseries as esta
+import emission.storage.decorations.analysis_timeseries_queries as esda
+
+import gzip
+import json
+import bson.json_util as bju
+
+def set_export_data(user_id):
+    time_query = epq.get_time_range_for_export_data(user_id)        # Does this have .startTs and .endTs atributes? Should we use those?
+    try: 
+        edp = ExportDataPipeline()
+        edp.user_id = user_id
+        edp.run_export_data_pipeline(user_id, time_query)
+        if edp.last_trip_done is None:
+            logging.debug("After run, last_trip_done == None, must be early return")
+        epq.mark_export_data_done(user_id, edp.last_trip_done)
+    except:
+        logging.exception("Error while exporting, timestamp unchanged")
+        epq.mark_export_data_failed(user_id)
```

Comment:
wrt your testing done, you should also confirm that the incremental export actually works - e.g.
- load data
- export #1
- load more data
- export #2

export #2 should not overlap with export #1

Reply:
Currently going through the final testing of the regiment of the edits I made this morning. Can you specify what you mean by t1 and t2? 
My idea for testing this would be to load in one test data file, run the intake pipeline, then load a new file and run it again to verify that the exported file has no overlap. Is this on par with what we are trying to test?

Reply:
`t1` and `t2` are typically used in the systems literature as placeholder timestamps. So they both represent arbitrary points of time, and `t1 < t2`.

>  My idea for testing this would be to load in one test data file, run the intake pipeline, then load a new file and run it again to verify that the exported file has no overlap. Is this on par with what we are trying to test?

That sounds fine. You should also test that the export is in fact exporting all the imported data.

Reply:
Sounds good, I will do this testing and push the commits. Would it be preferred if I open a new pull request or is this one fine?

Reply:
this is fine.